nextflow_schema.json

```diff
@@ -398,6 +401,9 @@
                 "skip_staramr": {
                     "type": "boolean",
                     "description": "Skip running StarAMR"
+                },
+                "skip_allele_calling": {
+                    "type": "boolean"
```

Comment:
Can you add a `description` here?

Reply:
Description added in: c8e0c8a003520471478e446d9fef5107084608ea